cmd/record.go

```diff
@@ -187,19 +192,14 @@ func (r *Record) GetCmd() *cobra.Command {
 			}
 
 			if isDockerCmd && len(path) > 0 {
-				curDir, err := os.Getwd()
-				if err != nil {
-					r.logger.Error("failed to get current working directory", zap.Error(err))
-					return err
-				}
 				// Check if the path contains the moving up directory (..)
 				if strings.Contains(path, "..") {
 					path, err = filepath.Abs(filepath.Clean(path))
 					if err != nil {
 						r.logger.Error("failed to get the absolute path from relative path", zap.Error(err), zap.String("path:", path))
 						return nil
 					}
-					relativePath, err := filepath.Rel(curDir, path)
+					relativePath, err := filepath.Rel("/files", path)
```

Comment:
I had to rebase since There was an update in the main repo before I pushed into my forked one. This change is not mine

Reply:
This is the commit for the above change :
https://github.com/keploy/keploy/commit/e28694504313fb127183a8fb5e1163d89a5723cb, 
Can i proceed ahead with the rest of review?

Reply:
We can't proceed without you removing these changes. In the future make sure you create a seperate branch when working on anything, otherwise with the main branch you are going to face these problems everytime.

Reply:
ok I'll keep in mind.. so how do you suggest I resolve this?